src/pages/index.tsx

```diff
@@ -1,118 +1,12 @@
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
-
-const inter = Inter({ subsets: ['latin'] })
+import Head from 'next/head';
 
 export default function Home() {
+  console.log('hello Team');
```

Comment:
❌ 
we should not leave conosle.logs in the code, so it is better to remove this